base_layer/core/src/chain_storage/async_db.rs

```diff
@@ -315,8 +313,8 @@ impl<'a, B: BlockchainBackend + 'static> AsyncDbTransaction<'a, B> {
         self
     }
 
-    pub fn insert_header(&mut self, header: BlockHeader, accum_data: BlockHeaderAccumulatedData) -> &mut Self {
-        self.transaction.insert_header(header, accum_data);
+    pub fn insert_header(&mut self, chain_header: ChainHeader) -> &mut Self {
```

Comment:
```suggestion
    pub fn insert_chain_header(&mut self, chain_header: ChainHeader) -> &mut Self {
```
This will be more clear what's going on

Reply:
Had to be renamed in a few places - but agree it's a little clearer, changed